docs/reST/ref/color.rst

```diff
@@ -214,6 +214,15 @@
       is useful if you want to unpack to r,g,b and not r,g,b,a. If you want to
       get the length of a Color do ``len(acolor)``.
 
+      .. note:: 
```

Comment:
You can check other deprecated functions to see how these are usually documented.

they use a tag like `.. deprecated:: 2.1.1` at the end. Should probably say DEPRECATED up top, rather than in a note.